### PR TITLE
AAP-12684 Add direct documentation feedback module

### PR DESCRIPTION
**Pull request by ariordan-redhat**
AAP-12684 

- Add direct documentation feedback module to the aap-common folder.
- Include the direct documentation module in all titles
- Backport to 2.4

Affects all titles.

### CODE REVIEW

**Comment by ccopelloRH** [comment]
I see one other edit: need to remove the original **Providing Feedback:** and this text: "If you have a suggestion to improve the documentation,...., please contact technical support...using the Docs component."

**Review by hherbly — APPROVED** [review]
Looks good to me too!

**Review by dcdacosta — APPROVED** [review]
LGTM and built locally with no errors and new module shows as expected.

File: downstream/titles/aap-operator-backup/docinfo.xml [hunk 1]
```diff
@@ -1,8 +1,11 @@
 <title>Red Hat Ansible Automation Platform Operator Backup and Recovery Guide</title>
 <productname>Red Hat Ansible Automation Platform</productname>
 <productnumber>2.4</productnumber>
-<subtitle>This guide provides procedures and reference information to backup and recover installations of the Red Hat Ansible Automation Platform operator on OpenShift Container Platform</subtitle>
+<subtitle>
+Safeguard against data loss with backup and recovery of Ansible Automation Platform operator on OpenShift Container Platform
+</subtitle>
 <abstract>
+This guide provides procedures and reference information to backup and recover installations of the Red Hat Ansible Automation Platform operator on OpenShift Container Platform
```
Comment (dcdacosta) [inline downstream/titles/aap-operator-backup/docinfo.xml:8]: Add a period at the end of this sentence.

File: downstream/titles/dev-guide/docinfo.xml [hunk 2]
```diff
@@ -1,8 +1,9 @@
 <title>Red Hat Ansible Automation Platform Creator Guide</title>
 <productname>Red Hat Ansible Automation Platform</productname>
 <productnumber>2.4</productnumber>
-<subtitle>This guide is intended for developers looking to learn how to use Ansible in creating content for automation.</subtitle>
+<subtitle>Learn to create automation content with Ansible</subtitle>
 <abstract>
+<para>This guide helps developers learn how to use Ansible to  create content for automation.</para>
```
Comment (dcdacosta) [inline downstream/titles/dev-guide/docinfo.xml:6]: Is there an extra space between to and create?

**Review by dcdacosta — APPROVED** [review]
A few minor nits, otherwise, LGTM!

**Review by hherbly — APPROVED** [review]

**Pull request merged**